server/apps/immich/src/api-v1/album/response-dto/album.ts

```diff
@@ -0,0 +1,30 @@
+import { AlbumEntity } from '../../../../../../libs/database/src/entities/album.entity';
+import { mapUser } from '../../user/response-dto/user';
+import { mapAsset } from '../../asset/response-dto/asset';
+import { Asset } from '../../asset/response-dto/asset';
+import { User } from '../../user/response-dto/user';
+
+export interface Album {
```

Comment:
Also, can we change the name to `AlbumResponseDto` just to make it explicit?

Reply:
yep 👍